cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -708,6 +711,8 @@ instance ( ConvertRawHash blk
          Just (_, _, _, s2 :: AF.AnchoredFragment (Header blk)) ->
            AF.toOldestFirst s2
          Nothing -> [] -- No sense to do validation here.
+     chainLengthΔ :: AF.AnchoredFragment (Header blk) -> AF.AnchoredFragment (Header blk) -> Int
+     chainLengthΔ = on (-) (fromWithOrigin (-1) . fmap (fromIntegral . unBlockNo) . AF.headBlockNo)
```

Comment:
I'm unsure about this `-1`, but it probably doesn't matter or at least only in the very minor scenario of only immediately starting a fresh node.

Reply:
@nfrisby, I think that since `BlockNo 0` is right after Genesis, it follows that:

`chainLengthΔ (..AF @ BlockNo 0) (..AF @Genesis)` -> `1` -- which _seems_ to make sense.

Maybe.

Reply:
Ah; OK. Since we're applying this function to both arguments of `-`, I think this is good. And your reasoning seems right to me for Cardano, which is what's relevant here.

So: LGTM!